src/styles/topbar.css

```diff
@@ -0,0 +1,50 @@
+.topbar {
+  width: 100%;
+  height: 380px;
+  position: sticky;
+  top: 0;
+  z-index: 999; }
+
+.topbarWrapper {
+  background-color: aquamarine;
+  height: 100%;
+  padding: 0px 20px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between; }
+
+.h3 {
+  font-weight: 600;
+}
```

Comment:
Style: Do we put the last `}` on a new line or not? Now the file uses both ways. I would vote for the h3 way.

Reply:
removing self coded css stylesheets in favour for tailwind autogenerated stylesheets in the future